python/amici/petab_objective.py

```diff
@@ -622,7 +623,8 @@ def _get_measurements_and_sigmas(
 def rdatas_to_measurement_df(
         rdatas: Sequence[amici.ReturnData],
         model: amici.Model,
-        measurement_df: pd.DataFrame) -> pd.DataFrame:
+        measurement_df: pd.DataFrame,
+        simulation: bool = False) -> pd.DataFrame:
```

Comment:
I'd change that to string argument `measurement_column` with default value `petab.C.MEASUREMENT`.
Gives some more flexibility.

Reply:
is there a way to set 2 "defaults" so users in interactive python can choose from a list, but still put in freehand?

Reply:
not aware of it, but that doesn't mean much

Reply:
you could create a second function `rdatas_to_simulation_df` that pipes all arguments to `rdatas_to_measurement_df` but has a different default for the column name